spec/integration/support/command_line_examples.rb

```diff
@@ -108,5 +108,20 @@
         end
       end
     end
+
+    describe "remove" do
+      context "when no arguments given" do
+        it "shows the help message for the remove command" do
+          message = /No arguments given. Nothing to do./
+          expect {
+            @machinery.run_command(
+              "#{machinery_command} remove",
+              as: "vagrant",
+              stderr: :capture
+            )
+          }.to raise_error(Pennyworth::ExecutionFailed, message)
+	end
```

Comment:
`end` at 123, 1 is not aligned with `it "shows the help message for the remove command" do` at 114, 8<br>Tab detected.